org.civicrm.reqangular/test/services/api.spec.js

```diff
@@ -185,11 +193,59 @@ define([
         });
       });
 
+      /**
+       * Mocks and sends a fake POST request
+       *
+       * @param  {any} returnValue - value to be returned by the POST request
+       * @param  {Object} params - params to be used in the API call
+       * @return {Promise}
+       */
       function expectAndSendPOST (returnValue, params) {
         $httpBackend.whenPOST('/civicrm/ajax/rest').respond(returnValue);
 
         return api.sendPOST(entity, action, params);
       }
     });
+
+    describe('getAll()', function () {
+      var returnValue = { is_error: 0, values: [{}] };
+
+      describe('custom options', function () {
+        beforeEach(function () {
+          spyOn(api, 'sendGET').and.returnValue(returnValue);
+          $rootScope.$digest();
+        });
+
+        describe('when no custom options are passed', function () {
+          beforeEach(function () {
+            api.getAll(entity, {});
+          });
+
+          it('default options are still used in the API call', function () {
```

Comment:
The description of the test should make a sentence with `it`.
Example: it - "still uses the default options in the API call".
Please check other instances.